hw/ip/otbn/doc/_index.md

```diff
@@ -499,6 +523,107 @@ This signal is high when OTBN is not running.
 The cycle after a write to {{< regref "CMD.start" >}}, the signal goes low.
 This remains low until the end of the operation (either from an [`ECALL`]({{< relref "isa#ecall" >}}) or an error, at which point it goes high again.
 
+### Data Integrity Protection {#design-details-data-integrity-protection}
+
+OTBN stores and operates on data (state) in its dedicated memories, register files, and internal registers.
+OTBN's data integrity protection is designed to protected all data stored and transmitted within OTBN from modifications through physical attacks.
+
+During transmission, the integrity of data is protected with an integrity protection code.
+Data at rest in the instruction and data memories is additionally scrambled.
```

Comment:
Is it worth clarifying here or elsewhere that it's the full data + integrity that gets scrambled and stored in memory?

Reply:
There is a more detailed description of that in the sections "Data Memory (DMEM) Integrity Protection" and "Instruction Memory (IMEM) Integrity Protection" below. This initial paragraph is only meant as a high-level intro. Let me know if the description further down is unclear or can be extended.